components/documents/privacy/PrivacyPolicyContent.vue

```diff
@@ -11,3 +11,9 @@ const { data } = await useAsyncData('/', () =>
     <ContentRenderer :value="data" />
   </PageContainer>
 </template>
+
+<style scoped>
+:deep(ul) {
+  @apply text-[#808080] list-[circle];
+}
+</style>
```

Comment:
Do you think we can apply the same result with `module` instead of `scoped`? just for consistency?

It's ok if it is not possible

Reply:
i'll try to see if it is possible

Reply:
done @kelsos